deploy/addons/ingress/ingress-dp.yaml.tmpl

```diff
@@ -49,7 +49,7 @@ spec:
       serviceAccountName: ingress-nginx
       containers:
         - name: controller
-          image: {{default "us.gcr.io/k8s-artifacts-prod/ingress-nginx" .ImageRepository}}/controller:v0.40.2
+          image: {{default "us.gcr.io/k8s-artifacts-prod" .ImageRepository}}/{{.Images.IngressController}}
```

Comment:
can we update all the addon images as part of this PR?

Reply:
Of course, just need more time.